client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html

```diff
@@ -38,6 +38,27 @@
                                 data-ng-bind="server.commercialRange !== 'housing' ? server.description || '-' : ('server_configuration_description_housing' | translate)"></dd>
                         </dl>
                     </div>
+                    <div class="oui-tile__item"
+                        data-ng-if="isEligibleForUpgrade() && upgradeName">
+                        <dl class="oui-tile__definition">
+                            <dt class="oui-tile__term">
+                                <span data-translate="server_configuration_upgrade_title"></span>
+                                <span class="badge" data-translate="server_configuration_upgrade_badge"></span>
```

Comment:
Is a `.oui-status` class could be used?

Reply:
Not here, too big, since it's in the title. But yes, I've tested it and it doesn't fit in.
We don't really have an equivalent for that with UI Kit v2.

I saw that the badge was already used in the others pages. So I take that in replacement.